manager/src/main/java/io/streamzi/openshift/API.java

```diff
@@ -131,24 +127,21 @@ public String getProcessorFlowDeployment(@PathParam("name") String name) {
     public void postFlow(String flowJson) {
         logger.info(flowJson);
         try {
-            ProcessorFlowReader reader = new ProcessorFlowReader();
-            ProcessorFlow flow = reader.readFromJsonString(flowJson);
+            ObjectMapper mapper = new ObjectMapper();
```

Comment:
in other methods (e.g. `getGlobalProperties()`) there is also an `ObjectMapper` - let's move it to class level - not method level